weave-js/src/components/Panel2/panellib/libcolors.ts

```diff
@@ -22,11 +19,14 @@ export const useColorNode = (inputNode: Node): NodeOrVoidNode => {
   return useMemo(() => {
     if (
       frame.runColors == null ||
-      isVoidNode(frame.runColors) ||
-      !isAssignableTo(
-        inputNode.type,
-        taggedValue(typedDict({run: 'run'}), 'any')
-      )
+      isVoidNode(frame.runColors)
+      // This was added by this pr https://github.com/wandb/weave/pull/865/files
+      // to fix a slow query, it also as a result broke run colors in charts
+      // I am going to comment out to fix run colors but we should take another look at this later
+      // || !isAssignableTo(
+      //   inputNode.type,
```

Comment:
out of curiosity, for the case that previously got caught in this check, what was the type of `inputNode`?

Reply:
for the case I was looking at it was all typedDict({project: project})